README.md

```diff
@@ -1,3 +1,58 @@
 # eslint-plugin-app-copy
 
 An ESLint plugin to enforce Geckoboard's in-house style guide for application copy.
+
+## Installation
+
+Assuming you already have ESLint installed in your project, simply run:
+
+`npm install --save-dev https://github.com/geckoboard/eslint-plugin-app-copy@latest`
+
+## Configuration
+
+Within your `.eslintrc` file you'll need to add this plugin to the plugins section, and add the additional rules:
```

Comment:
Do we also want to include a set of instructions for ESLint 9, which [uses `eslint.config.js`](https://eslint.org/docs/latest/use/migrate-to-9.0.0#flat-config)?

Reply:
I don't think we need to at this point, maybe when we test with eslint 9